DEVELOPMENT.md

```diff
@@ -1,40 +1,80 @@
 # Development
```

Comment:
@MikeMcC399 In our other repos, this type of document is typically named `CONTRIBUTING.md`. Mind updating to align on this pattern?

Reply:
@emilyrohrbough 
It's a good idea to standardize on `CONTRIBUTING.md`.

I suggest to do this in a follow-on PR, add a welcome / thank you message into the document, copy the [CODE_OF_CONDUCT.md](https://github.com/cypress-io/cypress/blob/develop/CODE_OF_CONDUCT.md) and link it, etc.